rs.json

```diff
@@ -107,9 +107,9 @@
   "inprogress": "U toku",
   "completed": "Završeno",
 
-  "today": "Dans",
+  "today": "današnji dan",
   "uptime": "uptime",
-  "daysAgo": "pre dana",
+  "daysAgo": "dana",
```

Comment:
Sytem Still says "90 dana" instead of "pre 90 dana".  It adds "daysago" to 90, instead it should do write 90daysago